### DISCUSSION TITLE
Changing the deployment image

### DISCUSSION THREAD

**Discussion by buaydin** [vmware-tanzu/velero#6002] [Community support Q&A] [answered]
Hello Guys,

We are using Nexus as docker image proxy. So the only way for us to download docker images is from docker hub using nexus repo url in the front of image name.
My question is how can I edit Deployment.yaml file before installation process starts or how can I build and create tar.gz file with my own. (after changing image name)

**Comment by ghost** [comment, marked as answer]
Hi @buaydin ,

If you are using a custom container image registry, you can follow the document below.

https://velero.io/docs/main/on-premises/

ex:
`velero install --image=$PRIVATE_REG/velero:$VELERO_VERSION --plugins=$PRIVATE_REG/velero-plugin-for-aws:$PLUGIN_VERSION <....>`
